pkg/crc/services/dns/dns.go

```diff
@@ -51,11 +51,6 @@ func setupDnsmasq(serviceConfig services.ServicePostStartConfig) error {
 	// Remove the dnsmasq container if it exists during the VM stop cycle
 	_, _, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f dnsmasq")
 
-	// Remove the CNI network definition forcefully
-	// https://github.com/containers/libpod/issues/2767
-	// TODO: We need to revisit it once podman update the CNI plugins.
-	_, _, _ = serviceConfig.SSHRunner.Run(fmt.Sprintf("sudo rm -f /var/lib/cni/networks/podman/%s", dnsContainerIP))
-
```

Comment:
The link suggests the CNI update is a 3.2 change, which would make it something fairly new as the current 4.8 bundles only have podman 3.0.1. It's been backported in https://github.com/containers/podman/commit/ce4265af83aca3a30909476b21b8f000a90011e3 aka `v3.0.0-rc3~3^2` so this should be fixed in at least all 4.8 bundles.

Reply:
Yes it is backported till  v3.0 so all bundles from 4.8 side should work because those are using 3.0.x or higher.